lib/lighthouse/veteran_verification/service.rb

```diff
@@ -77,12 +80,17 @@ def transform_response(response)
           VeteranVerification::Constants::NOT_FOUND_MESSAGE
         end
 
-      log_reason(reason)
+      log_not_confirmed(reason)
       response
     end
 
-    def log_reason(reason)
-      ::Rails.logger.info('Vet Verification Status Success', { not_confirmed_reason: reason })
+    def log_not_confirmed(reason)
+      ::Rails.logger.info('Vet Verification Status Success: not confirmed',
```

Comment:
Is there a reason you're using the leading `::` before Rails? I looked for other uses of it in the vets-api and found quite a few. I don't think it should ever be necessary because there should never be other instances of the `Rails` namespace.

Reply:
@AdamKing0126 @kpethtel Adding :: ensures that Ruby resolves Rails from the root namespace, ignoring any Rails constant that might exist in the current or a nested namespace. This eliminates any ambiguity and ensures you're always referring to the global Rails constant. Basically, they are ensuring they won't mess up anything.